seed/challenges/00-getting-started/getting-started.json

```diff
@@ -309,6 +309,81 @@
+          ""
+        ],
+        [
+          "//i.imgur.com/UVB9hxp.jpg",
+          "Un'immagine di un camper in un caffè che lavora ai progetti di freeCodeCamp.",
+          "Crediamo che gli esseri umani imparino meglio facendo. Quindi lavorerai per la maggior parte del tempo a dei progetti pratici. Ti daremo una lista di specifiche (Storie-Utente 'Agili'), e Tu dovrai decidere come sviluppare delle applicazioni che soddisfino queste specifiche.",
+          ""
+        ],
+        [
+          "//i.imgur.com/pbW7K5S.jpg",
+          "Un'immagine che mostra le nostre certificazioni front-end, back-end, e visualizzazione dei dati (400 ore ciascuno), i nostri progetti no-profit (800 ore), e la preparazione ai colloqui (80 ore) per un totale di 2.080 ore di esperienza.",
+          "Il nostro curriculum è diviso in 4 certificazioni. Queste certificazioni sono standardizzate, e immediatamente verificabili dai tuoi clienti come freelance e dai futuri datori di lavoro. Come tutto il resto di freeCodeCamp, queste certificazioni sono gratuite. Ti consigliamo di farle in ordine, ma non è obbligatorio. Per le prime tre certificazioni, ci vogliono 400 ore ciascuna, per la certificazione finale ci vogliono 800 ore, e questa prevede lo sviluppo di progetti reali per delle organizzazioni no-profit. ",
+          ""
+        ],
+        [
+          "//i.imgur.com/akGfLoa.jpg",
+          "Uno screenshot del nostro certificato di sviluppatore Front-End",
+          "Per ottenere la certificazione verificata di Sviluppatore Front-End, dovrai realizzare 10 progetti utilizzando HTML, CSS, jQuery, e JavaScript.",
+          ""
+        ],
+        [
+          "//i.imgur.com/ocEL0D3.jpg",
+          "Uno screenshot del nostro certificato di Visualizzazione Dati",
+          "Per ottenere la certificazione di Visualizzazione Dati, dovrai realizzare 10 progetti utilizzando React, Sass e D3.js.",
+          ""
+        ],
+        [
+          "//i.imgur.com/8HCk1Od.jpg",
+          "Uno screenshot del nostro certificato di sviluppatore Back-End",
+          "Per ottenere la certificazione di Sviluppatore Back-End, dovrai realizzare 10 progetti utilizzando Node.js, Express e MongoDB. Userai Git e Heroku per distribuirli in rete.",
+          ""
+        ],
+        [
+          "//i.imgur.com/yXyxbDd.jpg",
+          "Una schermata della nostra directory dei progetti no-profit.",
+          "Dopo aver ottenuto tutti e tre i certificati, potrai collaborare con un altro camper e utilizzerai metodologie di sviluppo 'Agili' per sviluppare due progetti reali per delle organizzazioni no-profit. Inoltre dovrai aggiungere funzionalità a due progetti no-profit sviluppati precedentemente. Quando avrai finito, avrai un portafoglio di applicazioni reali utilizzate quotidianamente.",
+          ""
+        ],
+        [
+          "//i.imgur.com/PDGQ9ZM.jpg",
+          "Un'immagine di alcuni campers che lavorano insieme ai progetti in un caffè a Seoul.",
```

Comment:
I think it should be: "lavorano insieme ai progetti in un **bar** a Seoul" o "in un **cafè** a Seoul". "Caffè" is the translation of "coffee" not "cafè"!